docs/topics/i18n/timezones.txt

```diff
@@ -397,6 +397,11 @@ will be set to :setting:`DATABASE-TIME_ZONE` or ``UTC`` respectively, so that Dj
 obtains correct datetimes in all cases. You don't need to perform any data
 conversions.
 
+.. note::
```

Comment:
Don't mind if this commit is included, or not. Happy to drop it if the merger likes.

Reply:
Can you change `note` to `admonition` with a title? We try to avoid [bare notes](https://docs.djangoproject.com/en/5.0/internals/contributing/writing-documentation/#guidelines-for-restructuredtext-files).